### ISSUE TITLE
Incorrect role is defined for the buttons present under 'Getting started' as Narrator announces them as table content in scan mode.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1376]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Turn On Narrator using 'Ctrl+Win+Enter'.
4. Navigate to the 'Get Profile' button using down arrow key.

**Actual Result:**
Incorrect role is defined for the buttons present under 'Getting started' as Narrator announces them as table content in scan mode.

**Expected Result:**
Correct role should be defined for the buttons present under 'Getting started' as Narrator should announce proper details.  

**User Impact:** 
Screen reader users will be impacted as they will get the incorrect information regarding the specified control.

**MAS Reference:**
[Name, Role, Value - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02)

**Additional context**
https://office.visualstudio.com/OC/_workitems/edit/5680729/

**Comment by ddyett** [comment]
@ElinorW what's the block on this one?

**Comment by ElinorW** [comment]
@ddyett I need to confirm this issue again with the accessibility team.
The conversation can be tracked here: https://office.visualstudio.com/OC/_workitems/edit/5680729/

**Comment by ElinorW** [comment]
This has been resolved now.
